### PR TITLE
New app: NASCAR Next Race

### PR DESCRIPTION
**Pull request by jvivona**
Shows Time date and location of Next NASCAR Race depending on user selection - Cup, Xfinity, Trucks

### CODE REVIEW

**Comment by jvivona** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/nascarnextrace/nascarnextrace.star [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+def main(config):
+    #TIme and date Information
+    location = config.get("location", DEFAULT_LOCATION)
+    loc = json.decode(location)
+    timezone = loc["timezone"]
+    now = time.now().in_location(timezone)
+    Year = now.format("2006")
+    series = config.get("NASCAR_Series", DEFAULT_SERIES)
+
+    NASCAR_DATA = http.get(NASCAR_API + series)
```
Comment (matslina) [inline apps/nascarnextrace/nascarnextrace.star:48]: Even with the ajcomputers API between Tidbyt and the original data source, you might want to add some caching. Keeps request rate down for the API, improves latency and also makes us more likely to render in the face of some networking glitch or say the API restarting, etc.

File: apps/nascarnextrace/nascarnextrace.star [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    "xfinity" : [ "xfinity", "#4427ad", "#fff", "NASCAR Xfinity Series" ],
+    "trucks" : [ "trucks", "#00669e", "#fff", "Camping World Truck Series" ]
+}
+"""
+
+def main(config):
+    #TIme and date Information
+    location = config.get("location", DEFAULT_LOCATION)
+    loc = json.decode(location)
+    timezone = loc["timezone"]
```
[the remark this answers is not in the document]
Comment (jvivona) [reply apps/nascarnextrace/nascarnextrace.star:43]: is there an example of doing it this way somewhere in the doc?    Frankly - I saw how some other apps were doing it and just did it their way.  would love to make it easier & 1 less thing.   do I literially use config.get(tz) in the starlark code?

**Comment by matslina** [comment]
Coolio. Merging.

One very minor note: there's no real need to base64 encode cache key/data. Any string can go in there as is, including binary data. But that's no blocker for merging!

**Pull request merged**